typings/index.d.ts

```diff
@@ -780,8 +674,8 @@ declare module 'klasa' {
 
 //#region Stores
 
-	export abstract class Store<K, V extends Piece, VConstructor = Constructable<V>> extends Collection<K, V> {
-		public constructor(client: KlasaClient, name: string, holds: V);
+	export abstract class Store<K, V extends Piece, VConstructor = Constructor<V>> extends Collection<K, V> {
+		public constructor(client: KlasaClient, name: string, holds: VConstructor);
 		public readonly client: KlasaClient;
 		public readonly holds: VConstructor;
```

Comment:
Is this wrong? Probably. But it stops a TS error when you try to KlasaClient#registerStore a custom store. 
```suggestion
		public readonly holds: V;
```

Reply:
I told you it's absolutely wrong, holds is a class, not an instance.